### DISCUSSION TITLE
[Esame 15 gennaio 2024] Esercizio 2

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#451] [Esami M-Z]
![15 gen es.2](https://github.com/user-attachments/assets/b175f9df-8e08-40fb-a463-19460a251280)

<details>
  <summary>soluzione del prof</summary>
  <br>
  <img src="https://github.com/user-attachments/assets/f44759b9-b2a7-4ff2-9164-d8a056275c02" alt="soluzione"  />
  <img src="https://github.com/user-attachments/assets/99d07417-2d49-49e6-9946-ae9ce577983c" alt="soluzione 2"  />

</details>

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/bf47ca1f-d6ec-49a7-8ad9-ade15157af57)

**Comment by niccolo-fato** [comment]
<img width="1280" height="1102" alt="image" src="https://github.com/user-attachments/assets/39b264ba-ef74-4325-a150-64d5be34a673" />
